src/apps/Permissions/EmptyFilteredPermissions.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
```

Comment:
Yeah, we definitely need an `aragonUI` component for this, we use it all around 😄